File: Examples/CSharp/Articles/CustomizingRibbonXML.cs
using System;
using System.IO;

using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    public class CustomizingRibbonXML
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            Workbook wb = new Workbook(sourceDir + "sampleCustomizingRibbonXML.xlsx");

            FileInfo fi = new FileInfo(sourceDir + "customUI_CustomizingRibbonXML.xml");
            StreamReader sr = fi.OpenText();
            wb.RibbonXml = sr.ReadToEnd();
            sr.Close();

            wb.Save(outputDir + "outputCustomizingRibbonXML.xlsx");

            Console.WriteLine("CustomizingRibbonXML executed successfully.");
        }
    }
}

File: Examples/VisualBasic/Articles/CustomizingRibbonXML.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Cells

Namespace Aspose.Cells.Examples.Articles
    Public Class CustomizingRibbonXML
        Public Shared Sub Main()
            ' The path to the documents directory.
            Dim dataDir As String = Aspose.Cells.Examples.Utils.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            Dim wb As New Workbook(dataDir & "aspose-sample.xlsx")
            Dim fi As New FileInfo(dataDir & "CustomUI.xml")
            Dim sr As StreamReader = fi.OpenText()
            wb.RibbonXml = sr.ReadToEnd()
            sr.Close()


        End Sub
    End Class
End Namespace